cmd/security-proxy-setup/res/docker/configuration.toml

```diff
@@ -45,7 +45,7 @@ WhiteList = "admin"
 Server = "edgex-vault"
 Port = 8200
 HealthCheckPath = "v1/sys/health"
-CertPath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
+CertPath = "/v1/secret/edgex/edgex-security-proxy-setup/"
```

Comment:
This looks to me like a backward incompatible change based on the recent discussions in Core WG about pathing in configuration. I assume such restrictions would apply to Vault paths as well. Thoughts @lenny-intel ?

Reply:
@tsconn23 Not sure that a change in the value is a breaking change. Also this is specific to this service which we deliver,  not a module or SDK. IMO, I think it is OK.